.gitignore

```diff
@@ -1,2 +1,9 @@
 .DS_Store
 node_modules
+
+# Jetbrains
+.idea_modules
+*.iml
+*.iws
+*.ipr
+.idea/
```

Comment:
Not related to npm tests or validation. I don't think Armor has strict standards around this yet. I think a bit of process overhead is worth having the PRs more focused. Don't need to move this now, I'm just using the soapbox to preach a bit.

Reply:
Good point. In the future, I'll be sure to keep the PRs as focused as possible.